tests/countries/test_saint_vincent_and_the_grenadines.py

```diff
@@ -0,0 +1,118 @@
+            "2017-01-02",
+            "2023-01-02",
+        )
+        self.assertHolidayName(f"{name} (observed)", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_national_workers_day(self):
+        name = "National Workers Day"
+        self.assertHolidayName(name, (f"{year}-05-01" for year in range(1979, 2050)))
+        dt = (
+            "2005-05-02",
+            "2011-05-02",
+            "2016-05-02",
+            "2022-05-02",
+        )
+        self.assertHolidayName(f"{name} (observed)", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_emancipation_day(self):
+        name = "Emancipation Day"
+        self.assertHolidayName(name, (f"{year}-08-01" for year in range(1979, 2050)))
+        dt = (
+            "2021-08-02",
+            "2027-08-02",
+        )
+        self.assertHolidayName(f"{name} (observed)", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_2024(self):
+        self.assertHolidays(
+            SaintVincentAndTheGrenadines(years=2024),
+            ("2024-01-01", "New Year's Day"),
+            ("2024-03-14", "National Heroes Day"),
+            ("2024-03-29", "Good Friday"),
+            ("2024-04-01", "Easter Monday"),
+            ("2024-05-01", "National Workers Day"),
+            ("2024-05-20", "Whit Monday"),
+            ("2024-07-08", "Carnival Monday"),
+            ("2024-07-09", "Carnival Tuesday"),
+            ("2024-08-01", "Emancipation Day"),
+            ("2024-10-27", "Independence Day"),
+            ("2024-12-25", "Christmas Day"),
+            ("2024-12-26", "Boxing Day"),
+        )
+
+    def test_l10n_default(self):
+        self.assertLocalizedHolidays(
+            ("2023-01-01", "New Year's Day"),
+            ("2023-01-02", "New Year's Day (observed)"),
+            ("2023-03-14", "National Heroes Day"),
+            ("2023-04-07", "Good Friday"),
+            ("2023-04-10", "Easter Monday"),
+            ("2023-05-01", "National Workers Day"),
+            ("2023-05-29", "Whit Monday"),
+            ("2023-07-10", "Carnival Monday"),
+            ("2023-07-11", "Carnival Tuesday"),
+            ("2023-08-01", "Emancipation Day"),
+            ("2023-10-27", "Independence Day"),
+            ("2023-12-25", "Christmas Day"),
+            ("2023-12-26", "Boxing Day"),
+        )
+
+    def test_l10n_en_us(self):
+        self.assertLocalizedHolidays(
+            "en_US",
+            ("2024-01-01", "New Year's Day"),
+            ("2024-03-14", "National Heroes Day"),
+            ("2024-03-29", "Good Friday"),
+            ("2024-04-01", "Easter Monday"),
+            ("2024-05-01", "Labor Day"),
+            ("2024-05-20", "Whit Monday"),
+            ("2024-07-08", "Carnival Monday"),
+            ("2024-07-09", "Carnival Tuesday"),
+            ("2024-08-01", "Emancipation Day"),
+            ("2024-10-27", "Independence Day"),
+            ("2024-12-25", "Christmas Day"),
+            ("2024-12-26", "Boxing Day"),
+        )
```

Comment:
`test_l10n_default` and `test_l10n_en_us` should tests the same year
```suggestion
    def test_l10n_en_us(self):
        self.assertLocalizedHolidays(
            "en_US",
            ("2023-01-01", "New Year's Day"),
            ("2023-01-02", "New Year's Day (observed)"),
            ("2023-03-14", "National Heroes' Day"),
            ("2023-04-07", "Good Friday"),
            ("2023-04-10", "Easter Monday"),
            ("2023-05-01", "Labor Day"),
            ("2023-05-29", "Whit Monday"),
            ("2023-07-10", "Carnival Monday"),
            ("2023-07-11", "Carnival Tuesday"),
            ("2023-08-01", "Emancipation Day"),
            ("2023-10-27", "Independence Day"),
            ("2023-12-25", "Christmas Day"),
            ("2023-12-26", "Boxing Day"),
        )
```